frontend/src/components/HomePage.js

```diff
@@ -26,11 +27,16 @@ export default class HomePage extends Component {
         lastOrderId: null,
         earnedRewards: 0,
         referralCode:'',
+        weblnEnabled: false
       }
     }
 
     setAppState=(newState)=>{
       this.setState(newState)
+      getWebln()
+        .then((webln) => {
+          this.setState({ weblnEnabled: webln !== undefined })
+        });
```

Comment:
Ok on that case let's keep it on the component using it (just the rewards), I move it there it was more convenient for future uses, anyways with the implementation of typescripts eventually it might be created a Context for this one-shot global data.